lib/src/modal_page/wolt_modal_sheet_page.dart

```diff
@@ -118,6 +118,10 @@ class WoltModalSheetPage {
   /// `true`.
   final bool? hasSabGradient;
 
+  /// Controls the draggability of the bottom sheet. This setting overrides the value provided
+  /// via [WoltModalSheet.show] specifically for this page when the modal is displayed as a bottom sheet.
+  final bool? enableDragForBottomSheet;
```

Comment:
Unless there is another `enableDrag` property in the package. I'd suggest just `enableDrag`.
The current name is too verbose IMO. The documentation should be enough to explain what enable drag is for.


See https://master-api.flutter.dev/flutter/material/BottomSheet/enableDrag.html

Reply:
Thanks, removed 868dd80ce11f3b3a5adacea958bbfa50c9c4d9ad and 219fc21b0822d8f94b739f6acbb7f005c66b3c80